__tests__/app.test.js

```diff
@@ -169,4 +169,77 @@ describe("App", () => {
         });
     });
   });
+
+  describe("POST - /api/articles/:article_id/comments", () => {
+    // Happy PATH 201 - Returns inserted object
+    it("Returns a 201 status code and a response object with a key of comment and the comment object that has been inserted", () => {
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send({ username: "butter_bridge", body: "This is a new comment" })
+        .expect(201)
+        .then(({ body }) => {
+          const { comment } = body;
+          expect(comment.article_id).toBe(2);
+          expect(comment.body).toBe("This is a new comment");
+          expect(comment.author).toBe("butter_bridge");
+          expect(comment.comment_id).toBe(19);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+    });
+    // 400 - Invalid input. No body provided.
+    it("Returns a 400 status code and a response object with a key of msg and a string 'No body or username key provided in the body of the request' as value", () => {
```

Comment:
We will also want to add another test for if searching for an invalid ID - for example articles/monkey
Giving a string when it wants a number - this would be a 400 error

Reply:
Done